src/models/dataobject.js

```diff
@@ -1,5 +1,62 @@
 import stampit from 'stampit';
 import {Meta, Model} from './base';
+import QuerySet from '../querySet';
+
+const DataObjectQuerySet = stampit().compose(QuerySet).methods({
+  /**
+    * Filters DataObjects.
+
+    * @memberOf QuerySet
+    * @instance
+
+    * @param {Object} filters
+    * @returns {QuerySet}
+
+    * @example {@lang javascript}
+    * DataObject.please().list({ instanceName: 'test-instace', className: 'test-class' }).filter({ field_name: { _contains: 'Lord Of The Rings' }}).then(function(dataobjects) {});
+
+    */
+  filter(filters = {}) {
+    this.query['query'] = JSON.stringify(filters);
+    return this;
+  },
+  /**
+    * Selects specified fields from DataObject.
+
+    * @memberOf QuerySet
+    * @instance
+
+    * @param {Object} fields
+    * @returns {QuerySet}
+
+    * @example {@lang javascript}
+    * DataObject.please().list({ instanceName: 'test-instace', className: 'test-class' }).fields(['title', 'author']).then(function(dataobjects) {});
+
+    */
+  fields(fields = []) {
```

Comment:
from what I know fields works for all endpoints :) maybe we should move this to base querySet?